Loop54.NetFramework/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="13.0.1" targetFramework="net462" />
   <package id="System.ValueTuple" version="4.5.0" targetFramework="net45" />
```

Comment:
Why not upgrade this as well?

Reply:
Oh, the targetFramework? I upgraded via VS and it didn't change this. I'm not sure what impact it has. https://docs.microsoft.com/en-us/nuget/reference/packages-config doesn't really explain it. I updated it manually now.